tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -1380,6 +1382,33 @@ TYPED_TEST(LinalgBackendEigenAllTypesTest, SGMatrix_mean)
 	EXPECT_NEAR(result, 4, get_epsilon<TypeParam>());
 }
 
+TYPED_TEST(LinalgBackendEigenAllTypesTest, SGMatrix_std_deviation_colwise)
```

Comment:
in a perfect world this would come in a different PR. But leave it in for now, just mentioning for next time

Reply:
Yup, sorry, I just needed an easy way to calculate the standard deviation